### DISCUSSION TITLE
Support for custom calendars / holidays

### DISCUSSION THREAD

**Discussion by vishnukvmd** [ente-io/ente#907] [Enhancements]
### Describe the feature
Provide customers with a way to choose from a list of additional calendars / holidays.

This could be surfaced within `Settings > General > Advanced > Calendars`.

### Reference point

https://github.com/ente-io/photos-app/blob/9cb49039ef4308ecb0a807857de197b9712ef1b8/lib/data/holidays.dart#L23

**Comment by mmjsmohit** [comment]
I would like to work on this issue. Can you please assign this to me?

**Comment by vishnukvmd** [comment]
Hey @mmjsmohit, on thinking more about this, we realized that there are a bit of UX nuances that might be hard for an external contributor to factor in (due to a lack of documentation from our side). Sorry about this, I'll update the tags 🙏
